### ISSUE TITLE
Replace Protocol Buffer Data Model with Pydantic

### ISSUE THREAD

**Issue by DavidOry** [AEP50/chandra-bot#24]
To make documentation easier via `mkdocs` and to remove the hassle of compiling the protocol buffer, use Pydantic for the data model.

**Comment by DavidOry** [comment]
@dhensle: are you still interesting in helping with this project? If so, this would be a helpful contribution for you to make.

**Comment by dhensle** [comment]
@DavidOry yes, still interested! And sounds good. I will try to make some progress in the coming weeks. Thanks!
